templates/data/kyma-operator-data.yaml

```diff
@@ -0,0 +1,28 @@
+templates:
+  - from: generic.tmpl
+    render:
+      - to: ../../prow/jobs/kyma-operator/operator.yaml
+        jobConfigs:
+          - repoName: kyma-project/kyma-operator
+            jobs:
+              - jobConfig:
+                  name: "pre-main-kyma-operator-lint"
+                  optional: "true"
+                  labels:
+                    preset-build-pr: "true"
+                    preset-kyma-guard-bot-github-token: "true"
+                    preset-gc-project-env: "true"
+                  command: "bash"
+                  args:
+                    - "-c"
+                    - "cd operator && golangci-lint run"
+                  env:
+                    GO11MODULE: "on"
+                  branches:
+                    - ^main$
+                inheritedConfigs:
+                  global:
+                    - jobConfig_default
+                    - jobConfig_presubmit
+                    - image_buildpack-golang
+                    - extra_refs_test-infra
```

Comment:
This line is not needed, as you aren't using any .sh script stored in the test-infra
```suggestion
```